pkg/cmd/experimental/bundlesecret/bundle-secret.go

```diff
@@ -0,0 +1,47 @@
+package bundlesecret
+
+import (
+	"encoding/base64"
+	"flag"
+	"fmt"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/util"
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+	"github.com/spf13/cobra"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+)
+
+// NewCmdBundleSecret creates a new bunde-secret command. The bundle-secret command will walk a given directory
+// and load its filenames into kubernetes secret keys with values that are base64 encoded
+func NewCmdBundleSecret(f *clientcmd.Factory, parentName, name string, out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "bundle-secret",
+		Short: "bundle data from an OpenShift dir to load kubernetes secret",
+		Long:  "bundle data from an OpenShift dir to load kubernetes secret",
+		Run: func(cmd *cobra.Command, args []string) {
+			//need to handle errors with certidir
+			certdir := flag.Arg(0)
+			filepath.Walk(certdir, walkpath)
+		},
+	}
+	return cmd
+}
+
+func walkpath(path string, info os.FileInfo, err error) error {
```

Comment:
walkpath recurses the entire subtree, we only want to do first level children right now.  it looks like you can return a special case SkipDir value to prevent it from recursing, we should look to do that.

also from a naming convention, it should be walkPath